.vscode/tasks.json

```diff
@@ -0,0 +1,12 @@
+{
+	// See https://go.microsoft.com/fwlink/?LinkId=733558
+	// for the documentation about the tasks.json format
+	"version": "2.0.0",
+	"tasks": [
+		{
+			"label": "Kill server",
+			"type": "shell",
+			"command": "kill --KILL (cat \"${workspaceFolder}/tmp/pids/server.pid\") && rm \"${workspaceFolder}/tmp/pids/server.pid\""
+		}
```

Comment:
Is there a situation where a simple `Ctrl + C` doesn't kill the server?

Reply:
Sometimes pids remains after unexpected closings. Happens to me sometimes when restarting my PC while the server is still on. I always fix it by manually removing the pids folder.

Reply:
> Is there a situation where a simple Ctrl + C doesn't kill the server?

I accidentally create an infinite loop while serving a request earlier when working on #259. Ctrl+C is handled, but the program refuses to shut down until all requests are handled. Because of the infinite loop, that would never be the case.

Maybe this was a very specific case though. Can't really think of other ones.